src/app/space/create/deployments/models/scaled-memory-stat.ts

```diff
@@ -1,39 +1,70 @@
+import { MemoryStat } from './memory-stat';
 import {
-  MemoryStat,
-  MemoryUnit
-} from './memory-stat';
+  MemoryUnit,
+  ordinal
+} from './memory-unit';
 import { ScaledStat } from './scaled-stat';
 
 import { round } from 'lodash';
 
 export class ScaledMemoryStat implements MemoryStat, ScaledStat {
 
-  private static readonly UNITS = ['bytes', 'KB', 'MB', 'GB'];
-
-  public readonly raw: number;
-  public readonly units: MemoryUnit;
+  private _raw: number;
+  private _units: MemoryUnit;
 
   constructor(
-    public readonly used: number,
-    public readonly quota: number,
-    public readonly timestamp?: number
+    private _used: number,
```

Comment:
Just in case anyone else comes across this wondering, it's because I've added those "getters" to the class (`get used(): number { }`). I've done that so that the static `from` function can still modify the internal state of an instance, since those properties are now private but not read-only, while callers from outside the class cannot see the private `_`-prefixed properties and so see them as read-only via the getters instead.